didman/api/v1/api.go

```diff
@@ -81,6 +82,7 @@ func (w *Wrapper) Preprocess(operationID string, context echo.Context) {
 	context.Set(core.StatusCodeResolverContextKey, w)
 	context.Set(core.OperationIDContextKey, operationID)
 	context.Set(core.ModuleNameContextKey, didman.ModuleName)
+	audit.Middleware(context, didman.ModuleName, operationID)
```

Comment:
Given the fact that controllers need to call the audit processing manually, it should no longer be called Middleware? It's no longer registered as middleware.

Reply:
renamed it to InitEchoContext